### DISCUSSION TITLE
Welcome to iconsole-android Discussions!

### DISCUSSION THREAD

**Discussion by haraldh** [haraldh/iconsole-android#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Arduan77** [comment]
Hi, are you still interested in developing applications supporting iConsole devices? I've never programmed for Android and I don't know much about Bluetooth but I tried to play around with it a bit. I have a treadmill with iConsole and I was able to download btsnoop_hci.log, besides I was able to decode the original iConsole+ application, maybe it will help you in further work? In the decoded program you can see a lot of information about the contents of packets.

**Reply by haraldh** [reply]
If you decoded the original application, we can try to do a clean room implementation. You provide the information about the packet format and the protocol, without revealing any code.

**Reply by Arduan77** [reply]
I sent you a link to the decoded program by e-mail, do you see anything interesting in it?

**Reply by haraldh** [reply]
@Arduan77 I am not allowed to answer :smile:
